src/libs/execution_time_estimator/execution_time_estimator.cpp

```diff
@@ -61,10 +61,14 @@ using Skill = ExecutionTimeEstimator::Skill;
  * @param skill The skill object to check.
  * @return true if this estimator can give an execution time estimate for the given skill.
  *
- * @fn std::pair<SkillerInterface::SkillStatusEnum, std::string> ExecutionTimeEstimator::execute(const Skill &skill) const
- * Let the estimator know that we are executing this skill, so it can apply
+ * @fn void ExecutionTimeEstimator::end_execute(const Skill &skill) const
+ * Let the estimator know that we start executing this skill, so it can apply
```

Comment:
not sure if the description is right here

Reply:
oops, that was for start_execute, fixed it now.